### DISCUSSION TITLE
can't view ubo list content

### DISCUSSION THREAD

**Discussion by ghajini** [uBlockOrigin/uBlock-issues#2189] [Q&A] [answered]
With Firefox android nightly 105.01a1 & stable 103
when i want to view ublock filters content by going into ubo dashboard=> filter lists => ublock filters => click eye menu to see contents=> results in blank page

![Screenshot_20220728-233221~2](https://user-images.githubusercontent.com/20338483/181606861-09c132ea-2ba3-46de-8e24-3d654ffa1eec.png)

is this firefox side bug ?
any thoughts @gorhill

**Comment by u-RraaLL** [comment]
I can't see any list's content. A new tab "opens", but not really. The tab count grows but I still see the filter list tab instead.

**Reply by peace2000** [reply]
I have this same issue as well. (I use Firefox Nightly).

**Comment by uBlock-user** [comment]
Specific to Android ? Working as expected on desktop.

**Reply by peace2000** [reply]
Yes, Android-only.

**Comment by Nojuuu** [comment]
Shouldn't this be converted into an issue?

**Reply by ghajini** [reply]
all discussions/issues are read by gorhill ,so i don't know if moving to an issues or creating new one makes any sense...when he feels /have time, he will fix

**Reply by MasterKia** [reply, marked as answer]
I have the same issue.
Edit:
`https://github.com/uBlockOrigin/uBlock-issues/issues/2271`
